docs/privacy/cspp/how-to-cspp.md

```diff
@@ -0,0 +1,103 @@
+buyer2$ dcrctl --wallet importxpub voting voting2-xpub
+```
+
+In addition, the mixed account xpub of the mixed ticket buyer must be imported by buyer1:
+
+```
+buyer2$ dcrctl --wallet getmasterpubkey mixed
+mixed-xpub
+buyer1$ dcrctl --wallet importxpub mixed mixed-xpub
+```
+
+The mixed ticket buying wallet may use the setup from the previous section. The old wallet must be configured sligtly differently:
+
+```
+buyer1$ dcrwallet --csppserver=cspp.decred.org:5760 \
+  --csppserver.ca=cspp.decred.org.pem \
+  --enableticketbuyer --purchaseaccount=default --mixedaccount=mixed/0 \
+  --ticketsplitaccount=default --changeaccount=unmixed \
+  --ticketbuyer.votingaccount=voting --mixchange
+```
+
+Note these differences:
+
++ `--mixedaccount=mixed/0` - The unmixed wallet must use the external (not internal) branch of the mixed account to avoid address reuseage problems arising from two wallets simultaneously deriving from the same branch.
++ `--ticketsplitaccount=default` - Unless this is set, the mixed account and branch will be used derive a fresh address for the mix. However, this would create issues when publishing a ticket, because the unmixed wallet does not have the required private key (mixed account is an imported xpub). This option must be set to a derived account with private keys, such as the purchasing source account.
+
+It may take large stakeholders up to the full ticket expiration of roughly 4.7 months to completely migrate to the new mixed wallet. Non-stakeholders can set these options, leaving out `purchaseaccount`, and participate in the mixing without buying tickets. To receive payments, generate new addresses from the unmixed account, then those payments will be mixed into the mixed account.
+
+Note that the mixing process requires the wallet being **unlocked** for longer periods of time, so that it can participate in a mix, which occurs every 20 minutes. Ticketbuyer wallets run unlocked already, but non-staking wallets will need to be left unlocked long enough to complete their mixes.
+
+---
+
+## Non-staking and change mixing
+
+Change outputs in the CoinJoin are not anonymous, and can easily be traced back to the set of inputs used during the mix. A dedicated unmixed account for CoinShuffle++ change is required, and it is not safe to spend change with other outputs in any transaction, including other mixes. To remedy this, dcrwallet provides a change mixing feature to create smaller mixed outputs of standard values and never submitting more than a single change output to the mixer in a request.
+
+These features are enabled with the following config:
+
+```
+$ dcrwallet --csppserver=cspp.decred.org:5760 \
+  --csppserver.ca=cspp.decred.org.pem \
+  --mixedaccount=mixed/1 --changeaccount=unmixed --mixchange
+```
+
+Alternatively, the mixaccount JSON-RPC may be used instead of the `--mixchange` option to mix single outputs from the account without leaving the wallet persistently unlocked.
```

Comment:
Sorry I missed this the first time - The name of the RPC `mixaccount` should also be in backticks